src/vm/runners/cairo_runner.rs

```diff
@@ -225,11 +227,20 @@ impl CairoRunner {
     }
     fn get_reference_list(&self) -> Vec<HintReference> {
         let mut references = Vec::<HintReference>::new();
+
         for reference in self.program.reference_manager.references.iter() {
             if let Some(register) = &reference.value_address.register {
                 references.push(HintReference {
                     register: register.clone(),
-                    offset: reference.value_address.offset,
+                    offset1: reference.value_address.offset1,
+                    offset2: reference.value_address.offset2,
+                })
+            } else {
+                // just a dummy HintReference to maintian the same lenght of vectors Vec<References> and Vec<HintReference>
+                references.push(HintReference {
+                    register: Register::FP,
+                    offset1: 0,
+                    offset2: 0,
```

Comment:
Maybe we could store a tuple with the index into `Vec<References>` and the `HintReference` instead of artificially increasing memory use.